autophonepulsemonitor.py

```diff
@@ -313,6 +313,7 @@ def handle_build(self, data, message):
             'comments',
             'packageUrl',
             'platform',
+            'revision',
```

Comment:
We will get this below when we call get_build_data. In fact we could probably reduce the "required" fields here but we can leave that for some other time.